src/packages/cli/scripts/install.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs')
+const path = require('path')
+const pkgUp = require('pkg-up')
+const Debug = require('debug')
+const debug = Debug('prisma:download')
```

Comment:
Looks like this file is mostly a renaming of "download.js"
Maybe we should also rename the debug log
```suggestion
const debug = Debug('prisma:install')
```